docs/sources/enterprise/access-control/enable-rbac-and-provisioning.md

```diff
@@ -63,6 +63,9 @@ This section describes how to enable RBAC by setting a feature flag or adding an
 
 ## Enable role provisioning
 
+You can create, change or remove [Custom roles]({{< relref "./roles.md#custom-roles" >}}) and create or remove [built-in role assignments]({{< relref "./roles.md#built-in-role-assignments" >}}), by adding one or more YAML configuration files in the [`provisioning/access-control/`]({{< relref "../../administration/configuration/#provisioning" >}}) directory.
```

Comment:
@mjseaman, how did this content get in here? The relrefs are broken and the "built-in" is now called "basic".